sycl/test-e2e/Matrix/SG32/joint_matrix_half.cpp

```diff
@@ -35,8 +35,8 @@ int main() {
   for (unsigned int i = 0; i < combinations.size(); i++) {
     if (combinations[i].nsize == 0) { // Intel AMX
       test<half, float, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16,
-           layout::row_major, 1>();
-      test<half, float, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16,
+           layout::ext_intel_packed, 2>();
```

Comment:
we should investigate why the VNNI transform does not work on AMX for this shape before we just remove it.

Reply:
// VNNI transform does not work for half yet (JIRA-Num)
``suggestion
layout::row_major, 1>();